README.md

```diff
@@ -169,134 +137,116 @@ Some changes do not require an RFC:
 If you're not sure whether your change requires an RFC, feel free to create an
 issue and ask.
 
-## What the process is
+## RFC Process
 
-In short, to get a major feature added to CDK/JSII, one usually first gets the
-RFC merged into the RFC repo as a markdown file. At that point the RFC is
-'ready' and may be implemented with the goal of eventual inclusion into
+In short, to get a major feature added to CDK/JSII, one usually writes an RFC 
```

Comment:
```suggestion
In short, to get a major feature added to CDK/jsii, one usually writes an RFC 
```

There are a few other instances of the upper-case JSII in the rest of the doc, if you're making this change.